include/overlay025/poketch_button.h

```diff
@@ -11,7 +11,7 @@ typedef void (* PoketchButtonCallback)(u32, u32, u32, void *);
 enum ButtonManagerState {
 	BUTTON_MANAGER_STATE_NULL = 0,
 	BUTTON_MANAGER_STATE_TOUCH,
-	BUTTON_MANAGER_STATE_OUT,
+	BUTTON_MANAGER_STATE_SLIDEOUT, // Slid touch off of button but did not release
```

Comment:
> Slid touch off of button but did not release

What does this mean?

Reply:
This is a specific state for when someone starts pressing a button on the touch screen, but then moves the stylus away from the button without lifting it off the screen.

Reply:
Maybe `BUTTON_MANAGER_STATE_DRAG` / `BUTTON_MANAGER_STATE_DRAGGING` then?

Reply:
Remove the comment.